### DISCUSSION TITLE
Livewire wire:end, nested component, not re-generated = Error, multiple root elements

### DISCUSSION THREAD

**Discussion by tanthammar** [livewire/livewire#3887] [7. Bugs (archived)]
I have a Livewire (form) component with a nested Livewire (input) component.

On initial page load I can see that both the parent and child component get their html `wire:end` comment.
When the **parent** component is updated (via updatedFoo() event hook), the **child** components `wire:end` comment is removed.

And I get the `Livewire: Multiple root elements detected. This is not supported` error.

I did force republish the Livewire assets, and cleared config, cache and views.

Livewire 2.6.7

**Comment by joshhanley** [comment]
@tanthammar I haven't had a chance to see if I can fix it yet, but it shouldn't impact the functioning of the app, as they are just console warnings.

When I get some time though I'll see if we can get it fixed.

Hope this helps!

**Comment by Tiagospem** [comment]
I have the same issue, however, since I use a UI for form elements, the code breaks.
